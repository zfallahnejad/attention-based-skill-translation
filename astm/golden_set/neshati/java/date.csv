22656,date
157882,date
260990,date
571407,date
992484,date
2970947,date
871026,date
1679863,date
829571,date
438154,date
3636601,date
1065197,date
