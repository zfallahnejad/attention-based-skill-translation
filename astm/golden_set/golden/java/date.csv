22656,160,113,0.5860805860805861
157882,58,24,0.7073170731707317
260990,51,69,0.425
571407,34,29,0.5396825396825397
992484,29,43,0.4027777777777778
2970947,26,26,0.5
871026,24,18,0.5714285714285714
829571,19,14,0.5757575757575758
1679863,19,9,0.6785714285714286
438154,18,16,0.5294117647058824
3636601,15,14,0.5172413793103449
1065197,13,8,0.6190476190476191
2783244,10,5,0.6666666666666666
331052,10,5,0.6666666666666666
